src/stage1/recurse.jl

```diff
@@ -852,7 +854,7 @@ function transform!(ci, meth, nargs, sparams, N)
     non_dce_finish!(compact)
     ir = complete(compact)
     ir = compact!(ir)
-    Core.Compiler.verify_ir(ir)
+    #Core.Compiler.verify_ir(ir)
```

Comment:
Should add an `allow_frontend_forms` argument to verify_ir that allows :opaque_closure_method in value position.

Reply:
cf https://github.com/JuliaLang/julia/pull/45786

Reply:
Something like this?
```suggestion
    if VERSION > v"1.9.0-DEV.854"
        Core.Compiler.verify_ir(ir; allow_frontend_forms=true)
    end
```

Reply:
not quite, but close.